SU2_CFD/src/driver_structure.cpp

```diff
@@ -262,29 +262,28 @@ CDriver::CDriver(char* confFile,
     
   }
 
+
   /*--- Definition of the interface and transfer conditions between different zones.
    *--- The transfer container is defined for zones paired one to one.
    *--- This only works for a multizone FSI problem (nZone > 1).
    *--- Also, at the moment this capability is limited to two zones (nZone < 3).
    *--- This will change in the future. ---*/
 
-  if ((rank == MASTER_NODE) && (fsi))
+  if ((rank == MASTER_NODE) && nZone > 1)
     cout << endl <<"------------------- Multizone Interface Preprocessing -------------------" << endl;
 
-
-
-  for (iZone = 0; iZone < nZone; iZone++) {
-    transfer_container[iZone] = new CTransfer*[nZone];
-    interpolator_container[iZone] = new CInterpolator*[nZone];
-    for (jZone = 0; jZone < nZone; jZone++) {
-      transfer_container[iZone][jZone] = NULL;
-      interpolator_container[iZone][jZone] = NULL;
+  if ( nZone > 1 ) {
+    for (iZone = 0; iZone < nZone; iZone++){
+      transfer_container[iZone] = new CTransfer*[nZone];
+      interpolator_container[iZone] = new CInterpolator*[nZone];
+      for (jZone = 0; jZone < nZone; jZone++){
+        transfer_container[iZone][jZone]     = NULL;
+        interpolator_container[iZone][jZone] = NULL;
+      }
     }
   }
-
-  if (((nZone > 1) && (nZone < 3)) && (fsi)) {
-    Interface_Preprocessing();
-  }
+  
+  Interface_Preprocessing();
```

Comment:
Are we obliged to use the function Interface_Preprocessing() for a single zone problem ?